packages/create-app/src/lib/versions.ts

```diff
@@ -94,6 +95,7 @@ export const packageVersions = {
   '@backstage/plugin-github-actions': pluginGithubActions,
   '@backstage/plugin-lighthouse': pluginLighthouse,
   '@backstage/plugin-org': pluginOrg,
+  '@backstage/plugin-permission-node': pluginPermissionNode,
```

Comment:
Also needs to be added as a peer dependency to create-app 😅, yeah this setup is weird